src/com/cap/omts/dao/AdminDAOImpl.java

```diff
@@ -51,7 +50,7 @@
 	@Override
 	public int addTheatre(Theatre theatre) throws OMTSException {
 		// TODO Auto-generated method stub
-		int insertedRows;

```

Comment:
please rename the variable appropriately. Here I suggest you to rename it as isInserted and initialize it to zero.